Tools/CISupport/ews-build/factories.py

```diff
@@ -194,6 +194,16 @@ def __init__(self, platform, configuration='release', architectures=None, remote
             self.addStep(RunJavaScriptCoreTests())
 
 
+class Test262BuildAndTestsFactory(Factory):
+    def __init__(self, platform, configuration='release', architectures=None, remotes=None, additionalArguments=None, runTests='true', **kwargs):
+        Factory.__init__(self, platform=platform, configuration=configuration, architectures=architectures, buildOnly=False, remotes=remotes, additionalArguments=additionalArguments, checkRelevance=True)
+        self.addStep(KillOldProcesses())
+        self.addStep(ValidateChange(addURLs=False))
+        self.addStep(CompileJSC(skipUpload=True))
+        if runTests.lower() == 'true':
```

Comment:
Why is this a string instead of a boolean?

Reply:
It's basically just a copy paste of `class JSCBuildAndTestsFactory(Factory):` a few lines above this one. The only thing I modified was `self.addStep(RunTest262Tests())` at the end instead of `self.addStep(RunJavaScriptCoreTests())` otherwise I didn't modify it any differently from what was already there.